docs/customization/api.rst

```diff
@@ -0,0 +1,80 @@
+Customizing API
+===============
+
+For create all endpoints in Sylius API we are using the API Platform.
+API Platform allows to configure endpoint by ``yaml`` and ``xml`` files or in ``annotations``.
+In these guide we will show you how to customize Sylius API endpoint with ``xml``.
+
+How to prepare project to customization?
+========================================
+
+For any kind of customization firstly you need to configure location of your configs.
```

Comment:
I would change `location` for `path` or `directory`. Location seems more suitable for navigation 😄